src/vs/platform/extensionManagement/node/extensionManagementService.ts

```diff
@@ -943,15 +943,19 @@ export class ExtensionManagementService extends Disposable implements IExtension
 	private _devSystemExtensionsPath: string | null = null;
 	private get devSystemExtensionsPath(): string {
 		if (!this._devSystemExtensionsPath) {
-			this._devSystemExtensionsPath = path.normalize(path.join(getPathFromAmdModule(require, ''), '..', '.build', 'builtInExtensions'));
+			// {{SQL CARBON EDIT}
+			let builtInPath = product.quality === 'stable' ? 'builtInExtensions' : 'builtInExtensions-insiders';
```

Comment:
I need to test this and see if it works, I'm a bit confused as to how it's meant to get handled